docs/docs/manual/grelfunctions.md

```diff
@@ -174,9 +174,19 @@ Returns a string converted to a number. Will attempt to convert other formats in
 
 Returns the array of strings obtained by splitting s by sep. The separator can be either a string or a regex pattern. For example, `"fire, water, earth, air".split(",")` returns an array of 4 strings: [ "fire", " water", " earth", " air" ]. Note that the space characters are retained but the separator is removed. If you include “true” for the preserveTokens boolean, empty segments are preserved.
 
-###### splitByLengths(s, n1, n2, ...) {#splitbylengthss-n1-n2-}
+###### splitByLengths(s, n1, n2, ...) [(optional) i1, i2] {#splitbylengthss-n1-n2-}
```

Comment:
```suggestion
###### splitByLengths(s, n1, n2, ...) {#splitbylengthss-n1-n2-}
```
This should be there. It has nothing specific about this function.

For the dash at the end of the hash, could you change it? It's going to require changes elsewhere...

Reply:
I'd say it's fine to leave the hash as it is…

Reply:
> I'd say it's fine to leave the hash as it is…

What's the logic @wetneb?
- [ ] It's autogenerated, and as such, it's extra work to tweak it.
- [ ] It's there to show that other param can optionnally come.
- [ ] Just not worth it, user doesn't really see it.

Regards, Antoine